docs: added custom logger info (#10108)
vitejs/vite: docs/config/index.md

})
```

> ## Config Custom Logger

Reviewer: ```suggestion
## Custom Logger
```

> Vite provides a `createLogger` function using which you can filter out warnings or provide a custom messages to your errors.

Reviewer: ```suggestion
Vite provides a `createLogger` function which you can filter out warnings or provide a custom message for your errors.
```
